src/components/storefront/StorefrontRouter/StorefrontRouter.js

```diff
@@ -25,19 +26,20 @@ const StorefrontRouter = () => {
     <ChakraProvider theme={storeTheme}>
       <Flex direction="column" minHeight="100vh">
         <Navbar />
-        <Container maxW="container.xl" flex="1">
```

Comment:
Originally @patrick-du added in this container but it didn't really fit well with my Cart page so I moved it inside `Home` and `Product`. I didn't put it inside `PageNotFound` since it looks the same anyways.

Reply:
After merging in your changes from #54, I still have the Containers inside `Home` and `Product`. I can probably move it up the component tree in a future PR or something